src/System.Buffers/src/System/Buffers/ArrayPoolEventSource.cs

```diff
@@ -22,7 +21,7 @@ internal enum BufferAllocationReason : int
             PoolExhausted
         }
```

Comment:
Yes, No one should be using GUIDs in their EventSource.   
Did someone actually make these events Localized?    Normally you do not do this.  

If FxResources... does not have any values of the form event_* task_*  keyword_*  then the LocalizationResources is junk.

Note that even if LocalizationResource does point to valid resources, I don't know of any tools that use it (only the windows event logger uses it and that only works if you register a manifest which we don't do, so mostly localization is dubious 

I would recommend removing the localization for now unless we have a true end-to-end scenario for it.